### PR TITLE
Home page revised

### PR DESCRIPTION
**Pull request by Kaushan05**
Home page is done

### CODE REVIEW

**Review by HarishDass — APPROVED** [review]

**Review by aarthi1246 — APPROVED** [review]

File: src/app/app-routing.module.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ const routes: Routes = [
       import('./attendance/attendance.module').then((m) => m.AttendanceModule),
   },
   {
-    path: 'EmployeeDetails',
+    path: 'Employee Details',
```
Comment (bharathidhasan2002) [inline src/app/app-routing.module.ts]: Remove space

File: src/app/home/home.component.ts [hunk 2]
```diff
@@ -1,14 +1,24 @@
...
 @Component({
   selector: 'app-home',
   templateUrl: './home.component.html',
   styleUrls: ['./home.component.css'],
 })
 export class HomeComponent {
-  value: any;
+  cardData: Carddetailsinarr[] = [
+    {
+      cardname: 'Employee Details',
```
Comment (bharathidhasan2002) [inline src/app/home/home.component.ts]: remove space

File: src/app/shared-module/carddetailsinarr.ts [hunk 3]
```diff
@@ -0,0 +1,3 @@
+export interface Carddetailsinarr {
+  cardname: string;
```
Comment (bharathidhasan2002) [inline src/app/shared-module/carddetailsinarr.ts]: use case sensitive on name and interface

File: db.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
+{
+  "posts": [
```
Comment (bharathidhasanm2s) [inline db.json]: what is the purpose of db.json

File: src/app/home/home.component.ts [hunk 5]
```diff
@@ -1,15 +1,26 @@
...
+    {
+      cardName: 'Attendance',
+    },
+  ];
+
   constructor(private router: Router) {}
+
   navigateTO(module: any) {
-    this.router.navigate([module]);
+    const joinName = module.split(' ').join('');
```
Comment (bharathidhasanm2s) [inline src/app/home/home.component.ts]: Can you use another method

File: src/app/shared-module/carddetailsinarr.ts [hunk 6]
```diff
@@ -0,0 +1,3 @@
+export interface CardDetailsinarr {
```
Comment (bharathidhasanm2s) [inline src/app/shared-module/carddetailsinarr.ts]: change the CardDetailsinarr to CardDetails

**Comment by Kaushan05** [comment]
Screenshot 2023-09-26 124914

**Comment by Kaushan05** [comment]
Screenshot 2023-10-04 104839
Screenshot 2023-10-04 104938

File: src/app/home/home.component.ts [hunk 7]
```diff
@@ -1,15 +1,26 @@
...
+    {
+      cardName: 'Attendance',
+    },
+  ];
+
   constructor(private router: Router) {}
+
   navigateTO(module: any) {
-    this.router.navigate([module]);
+    const joinName = module.replace(' ', '');
```
Comment (bharathidhasanm2s) [inline src/app/home/home.component.ts]: don't use extra variable name

**Review by bharathidhasanm2s — APPROVED** [review]

**Review by bharathidhasan2002 — APPROVED** [review]

**Pull request merged**